src/app/routes/api/misc.py

```diff
@@ -12,6 +12,7 @@
 from Crypto.PublicKey import RSA
 from Crypto.Hash import SHA256
 
+from ...routes.api.gmail import confirmation_email
```

Comment:
This import won't work for all Python interpreters.

Reply:
when I add gmail directly or ...api.email the file gives me an error saying unable to import file